ext/RMagick/extconf.rb

```diff
@@ -130,7 +130,11 @@ def configure_compile_options
 
       end
 
-      $CPPFLAGS << (have_macro('__GNUC__') ? ' -std=gnu99' : ' -std=c99')
+      $CPPFLAGS << if have_macro('__GNUC__')
+                     ' -std=gnu99 -Wno-void-pointer-to-int-cast -Wno-void-pointer-to-enum-cast -Wno-pointer-to-int-cast'
```

Comment:
Is this disabling warnings?

Reply:
You're right.
I added the flag because of compile-time warnings that cannot be removed and it become noise.